anitya/lib/backends/cpan.py

```diff
@@ -48,6 +48,23 @@ def get_version(cls, project):
         '''
         return cls.get_ordered_versions(project)[-1]
 
+    @classmethod
+    def get_version_url(cls, project):
+        ''' Method called to retrieve the url used to check for new version
+        of the project provided, project that relies on the backend of this plugin.
+
+        Attributes:
+            project (:obj:`anitya.db.models.Project`): Project object whose backend
+                corresponds to the current plugin.
+
+        Returns:
+            str: url used for version checking
+        '''
+        url = 'https://metacpan.org/release/%(name)s/' % {
```

Comment:
I think it would be nice to have some consistency in the use of the `%` format operator and `format()` method. Also if anitya is Python 3 only and runs on Python 3.6 f-strings are nice.